packages/api/src/util/lcd.ts

```diff
@@ -0,0 +1,50 @@
+import fetch from 'node-fetch'
+import { resolve } from 'url'
+
+export default class LCDClient {
+  private _endpoint: string
+
+  constructor(endpoint: string = "http://localhost:1317") {
```

Comment:
is it really necessary?
```suggestion
  constructor(endpoint: string) {
```

Reply:
I removed the other one from the parent but this one is good to have a default parameter I think